docs-markdown/src/controllers/template-controller.ts

```diff
@@ -14,6 +15,11 @@ export function applyTemplateCommand() {
 
 export function applyTemplate() {
     reporter.sendTelemetryEvent("command", { command: telemetryCommand });
-    vscode.commands.executeCommand("applyTemplate").then(
-        (err) => vscode.window.showErrorMessage("Docs-article-templates extension is not installed or disabled."));
+    const extensionName = "docsmsft.docs-article-templates";
+    const { msTimeValue } = generateTimestamp();
+    const friendlyName = "docsmsft.docs-article-templates".split(".").reverse()[0];
```

Comment:
not that it matters too much but you could:

`const [ friendlyName ] = "docsmsft.docs-article-templates".split(".").reverse();`

Reply:
Thanks @wibjorn , noted and will add to the cleanup list.